BMM.Core/GuardedActions/TrackOptions/PrepareTrackOptionsAction.cs

```diff
@@ -46,6 +52,14 @@ public class PrepareTrackOptionsAction :
         private readonly ISleepTimerService _sleepTimerService;
         private readonly IFirebaseRemoteConfig _firebaseRemoteConfig;
         private readonly IAnalytics _analytics;
+        private readonly IMediaPlayer _mediaPlayer;
+
+        private readonly List<decimal> _availablePlaybackSpeed = new()
+        {
+            0.75m,
```

Comment:
is 1.00m not an option?

Reply:
Right now the app will show additional "Default" option if user changed playback speed before. This resets speed to 1x.

But we can follow your idea, and get rid of this "Default" option, and always show the same set:

- 0.75x
- 1x
- 1.25x
- 1.5x

regardless if user selected anything before. 

What more convincing to you

Reply:
1x option should be called Normal (like YouTube)